lms/djangoapps/discussion_api/views.py

```diff
@@ -213,8 +214,8 @@ class CommentViewSet(_ViewMixin, DeveloperErrorViewMixin, ViewSet):
     """
     **Use Cases**
 
-        Retrieve the list of comments in a thread, create a comment, or modify
-        an existing comment.
+        Retrieve the list of comments in a thread, create or delete comment, or
```

Comment:
Actually, I think it would be best as "Retrieve the list of comments in a thread, create a comment, or modify or delete an existing comment.